spec/models/user_spec.rb

```diff
@@ -5,9 +5,54 @@
   include TestFactories
 
```

Comment:
overall in ur tests u dont always need to create and save it to the db, slows down ur tests, sometimes u can just do like user = User.new instead of User.create ... or create(:user) -- do some research on that today and figure out when its best to create vs just instantiate an object. Also there is mock_model and other ways to make an object.